SimG4CMS/Calo/src/HFGflash.cc

```diff
@@ -233,8 +233,11 @@ std::vector<HFGflash::Hit> HFGflash::gfParameterization(const G4Step* aStep, boo
   z1 *= 9.76972e-01 - 3.85026e-01 * std::tanh(1.82790e+00 * std::log(energy) - 3.66237e+00);
   p1 *= 0.96;
 
+#ifdef EDM_ML_DEBUG
+  G4int nSpots_sd = 0;  // count total number of spots in SD
+#endif
+
```

Comment:
@aandvalenzuela , I think `nSpots_sd` can be deleted. I do not see it is in use any where

Reply:
oops, forget it. I see it is used :-)